spec/stubs.rb

```diff
@@ -1,5 +1,3 @@
-require "securerandom"
```

Comment:
Moved this from here to the main lib because @krististepanek had some issue running payment intent create